idea/testData/inspectionsLocal/redundantEmptyClassInitializer/notEmpty2.kt

```diff
@@ -0,0 +1,6 @@
+// PROBLEM: none
+class Test {
+    <caret>init {
+        // comment
```

Comment:
Seems we can report the warning in this case because block is still redundant (comment can be moved to class body)